cmd/hw003/array/average/average.go

```
@@ -0,0 +1,23 @@
+package average
+
+import (
+	"log"
+	"strconv"
+	"strings"
+)
+
+func FindAver(line string) float64 {
+	var (
+		sum, count int
+	)
+	ar := strings.Split(line, ",")
+	for i, val := range ar {
+		n, err := strconv.Atoi(val)
+		if err != nil {
+			log.Fatal(err)
```

Your function is only responsible for some calculations and should not make decisions about program execution. So in this case it is better to return an error to the calling function.